draft-ietf-ppm-dap.md

```diff
@@ -451,25 +454,28 @@ Measurement:
   {{!VDAF}}.
 
 Minimum batch size:
-: The minimum number of reports in a batch.
+: The minimum number of reports that must be aggregated before a batch can be
+  collected.
 
 Public share:
-: The output of the VDAF sharding algorithm broadcast to each of the
+: The output of the VDAF sharding algorithm transmitted to each of the
   Aggregators. As defined in {{!VDAF}}.
 
 Report:
 : A cryptographically protected measurement uploaded to the Leader by a Client.
   Comprised of a set of report shares.
 
-Report Share:
-: An encrypted input share comprising a piece of a report.
+Report share:
+: An input share encrypted under the HPKE public key of an Aggregator.
```

Comment:
Pedantically, the report share includes a few additional pieces of information -- report metadata & public share.